server/routes/api/contacts/index.js

```diff
@@ -96,7 +96,7 @@ router.get('/:id', isAuthenticated, (req, res) => {
     });
 });
 
-router.put('/:id', isAuthenticated, (req, res) => {
+router.put('/edit/:id', isAuthenticated, (req, res) => {
```

Comment:
Is there a reason this was changed to /edit/:id?

Reply:
incorrectly thought that was the correct format